### PR TITLE
Add blob with sample text

### PR DESCRIPTION
**Pull request by alicjamakowska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,14 +21,15 @@
...
+        
 
     <link rel="manifest" href="favicon/manifest.json">
     <meta name="msapplication-TileColor" content="#167fbc">
     <meta name="msapplication-TileImage" content="favicon/ms-icon-144x144.png">
     <meta name="theme-color" content="#167fbc">
 </head>
 
-<body style="font-family:'Lato'">
+<body class="body-background" style="font-family:'Lato'">
```
Comment (michudzik) [inline index.html]: Avoid using inline styles :)

File: index.html [hunk 2]
```diff
@@ -37,9 +38,19 @@ <h1 class="page-title m-0"><b>Dzień Wydziału</b></h1>
         </section>
         <section id="countdown" class="font-weight-bold text-center my-2 counter align-middle"></section>
     </header>
-    <div><img src="./assets/wave.svg" class="w-100"/></div>
+    <div><img src="./assets/wave.svg" class="w-100" /></div>
+    <div class="d-flex w-100 justify-content-center">
+        <div class="d-flex justify-content-start blob-container text-white text-start position-relative">    
+            <img src="./assets/blob.svg" class="blob"/>
+            <div class="text-left font-weight-bold blob-font centered">
+                Tutaj kolejny tekst wędeczka oj bardzo
```
Comment (michudzik) [inline index.html]: WUT XD

File: main.css [hunk 3]
```diff
@@ -22,5 +26,25 @@
...
-    font-size: 32px;
+    font-size: 40px;
+}
+
+.blob {
+    width: 35%;
+}
+
+.blob-font {
+    font-size: 2.5vw;
```
Comment (michudzik) [inline main.css]: Why some font-sizes are defined using px and other using vw?

File: main.css [hunk 4]
```diff
@@ -1,16 +1,20 @@
 :root {
     --bg-blue: #4E81E0;
     --bg-counter: #4873C5;
```
Comment (michudzik) [inline main.css]: What colour is named `counter`?

File: index.html (hunk 2 above)
Comment (alicjamakowska) [reply index.html]: What do you mean?

File: main.css (hunk 4 above)
Comment (alicjamakowska) [reply main.css]: This is name of background for counter

File: main.css [hunk 5]
```diff
@@ -22,5 +27,122 @@
...
+    }
+    .centered {
+        position: absolute;
+        top: 50%;
+        left: 45%;
+        transform: translate(-50%, -50%);
+    }
+    .countdown-margin {
+        margin-top: 5px;
+    }
```
Comment (mkidawa) [inline main.css]: That class is only used with counter, therefore shouldn't margin property be included in .counter class?

File: main.css [hunk 6]
```diff
@@ -1,16 +1,21 @@
 :root {
     --bg-blue: #4E81E0;
     --bg-counter: #4873C5;
-    /* --bg-grey: #231F20; */
+    --bg-grey: #231F20;
+}
+
+.body-styling {
+    background-color: var(--bg-grey);
```
Comment (mkidawa) [inline main.css]: Body is unique tag so instead of class you can target it in css by reffering to "body".

**Pull request merged**